Duplicati/Library/Utility/FilterGroups.cs

```diff
@@ -554,6 +554,35 @@ private static IEnumerable<string> CreateLinuxFilters(FilterGroup group)
             }
         }
 
+        /// <summary>
+        /// Creates a filter for a special folder
+        /// </summary>
+        /// <param name="specialFolder">Special folder</param>
+        /// <returns>Special folder filter</returns>
+        private static string CreateSpecialFolderFilter(Environment.SpecialFolder specialFolder)
+        {
+            string folderPath = Environment.GetFolderPath(specialFolder);
+            if (!string.IsNullOrEmpty(folderPath))
+            {
+                string filter = FilterGroups.CreateWildcardFilter(folderPath);
+
+                // Duplicati matches filters against folder paths exactly.
+                // Meaning a filter for 'C:\Windows' won't match 'C:\Windows\'.
+                // So this makes sure special folder's filter's have a trailing directory separator.
+                // (Alternatively, this could append '*' to all folder filters.)
+                if (filter[filter.Length - 1] != System.IO.Path.DirectorySeparatorChar)

```

Comment:
Would the [Utility.AppendDirSeparator](https://github.com/duplicati/duplicati/blob/master/Duplicati/Library/Utility/Utility.cs#L425) method be useful here?

Reply:
Yes, I believe it would. I didn't notice that utility method previously.